.gitattributes

```diff
@@ -0,0 +1,10 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Declare files that will always have LF line endings on checkout.
```

Comment:
Not sure that I like this in gitattributes. We have users on Mac, Linux and Windows. Pretty sure we leave this to be always auto.  OK for now, but will check with the team.

Reply:
I think this is the right thing to do - we do the same in other projects. This will effectively only be required in Windows systems for bash scripts, etc. as UNIX/Linux will already checkout using LF for line endings.

Reply:
Sounds good. As long as it's consistent with other projects.

Thanks for the follow up.